docs/features/externaltools/h5p.md

```diff
@@ -2,45 +2,66 @@
 
-1. **Memory** You can either set the pictures to be the same or the match different pictures. In the first case the participant will have to find pairs. In the second case the participant has to find matching pictures. As long as the participant completes the memory game, full points will be rewarded. An example looks like this:  ![Memory](../img/externaltools/h5p/memory.png)
-2. **Find the words** In this quiz you can define a list of words that will be hidden in a field of letters. This will be done automatically. The words will appear in any possible way and direction in the letter field. It is possible for a word to be backwards diagonal. There are no penalties so the full points will be rewarded for finding all words. An example:  ![Find the words](../img/externaltools/h5p/find_the_words.png)
-3. **Drag and Drop** In this quiz the participants drop words either on marked areas over a picture or in question fields. They are not as easy to set up as the others. However it is more flexible than the other quizzes. It is possible to define which words can be dropped into which zones. Additionally the number of words per zone can be set as well. This makes the following examples possible:  ![Drag and Drop 1](../img/externaltools/h5p/drag_and_drop_1.png)  ![Drag and Drop 2](../img/externaltools/h5p/drag_and_drop_2.png)
-4. **Image Sequencing** This quiz allows you to set pictures in a correcter order. In will then mix them up for the participants. This is easy to set up and use. The example from the course looks like this:  ![Image sequencing](../img/externaltools/h5p/image_sequencing.png)
+1. **Memory** You can either set the pictures to be the same or to match different pictures. In the first case the participant will have to find pairs. In the second case the participant has to find matching pictures. As long as the participant completes the memory game, full points will be rewarded. An example looks like this: 
+   ![Memory](../img/externaltools/h5p/memory.png)
+ 
+3. **Find the words** In this quiz you can define a list of words that will be hidden in a field of letters. This will be done automatically. The words will appear in any possible way and direction in the letter field. It is possible for a word to be backwards diagonal. There are no penalties so the full points will be rewarded for finding all words. An example:
+  ![Find the words](../img/externaltools/h5p/find_the_words.png)
+  
+5. **Drag and Drop** In this quiz the participants drag and drop words either on marked areas over a picture or in question fields. They are not as easy to set up as the others. However it is more flexible than the other quizzes. It is possible to define which words can be dropped into which zones. Additionally the number of words per zone can be set as well. This makes the following examples possible:  
+
```

Comment:
Ich bin mir nicht ganz sicher wie wir die Einrückungen umsetzten sollen. Bei dir sind die Bilder immer in neuen Zeilen und habe eine leere Zeile darüber und darunter. Da bei mir nicht das doppelte Leerzeichen funktioniert hat, habe ich sie in die nächste Zeile gepackt. Von mir aus können die auch separate Zeilen bekommen aber ich würde die Leerzeilen rauslöschen.

Reply:
Ich wollte halt vermeiden dass die Bilder in line angezeigt werden und am liebsten auch ein wenig spacing um die Bilder rum. Vielleicht kennt unser Fontfuchs da eine Lösung? @T4rikA

Reply:
Weiß nicht ob das hier geht, aber man kann HTML in Markdown packen, dann vllt einfach mit `float: left` oder so

Reply:
Würde das layouting aber ganz am Ende machen?